src/pages/components/tree-view/usage.mdx

```diff
@@ -0,0 +1,356 @@
+  caret icon bounding box.
+
+<Row>
+<Column colLg={8}>
+
+![Example showing click target areas for a branch node](images/tree-view-usage-7.png)
+
+</Column>
+</Row>
+
+#### Selecting
+
+- To select a branch node the user can click anywhere on the node excluding the
+  caret icon bounding box.
+- To select a leaf node the user can click anywhere on the node container.
+
+<Row>
+<Column colLg={8}>
+
+![Example showing click target areas for selecting a node](images/tree-view-usage-8.png)
+
+</Column>
+</Row>
+
+#### Keyboard controls
+
+| Key           | Interaction                                                                                                                                 |
+| ------------- | ------------------------------------------------------------------------------------------------------------------------------------------- |
+| _Right arrow_ | When focus is on a closed node, opens the node; focus does not move.                                                                        |
+|               | When focus is on an open node, moves focus to the first child node.                                                                         |
+|               | When focus is on a leaf node, does nothing.                                                                                                 |
+| _Left arrow_  | When focus is on an open node, closes the node.                                                                                             |
+|               | When focus is on a child node that is also either a leaf node or a closed node, moves focus to its parent node.                             |
+|               | When focus is on a top level node that is also either a leaf node or a closed node, does nothing.                                           |
+| _Down arrow_  | Moves focus to the next node that is focusable without opening or closing a node.                                                           |
+| _Up arrow_    | Moves focus to the previous node that is focusable without opening or closing                                                               |
+| _Enter_       | Activates a node. In single-select trees where selection does not follow focus, the default action is typically to select the focused node. |
+
+## Modifiers
+
+### Node icons
+
+You can add node icons to visually represent and support a written node label.
+We recommend having consistent icon usage for all branch and leaf nodes. Make
+sure the icons accurately represent each node that they are related to. Using a
+folder icon for branch nodes and a document icon for leaf nodes is a commonly
+recognized pairing of icons used in tree view structures. If individual node
+data cannot be identified, do not display icons. When in doubt, use nodes
+without icons.
+
+<Row>
+<Column colLg={8}>
+
+![Example of tree view with node icons](images/tree-view-usage-9.png)
```

Comment:
I noticed you are using the same image for the `Overview` section and the `Modifier/Node icon` section. I think instead of repeating the image we can mock up a different one here in either of these places.

These are some of the products I know of that have some different examples:
— Cognos Analytics: https://ibm.ent.box.com/folder/155993052376
— EPM: https://ibm.ent.box.com/file/917294933176
— Treeview screenshots: https://ibm.ent.box.com/file/993258467435